test/jasmine/tests/hover_label_test.js

```diff
@@ -4601,6 +4601,74 @@ describe('hovermode: (x|y)unified', function() {
             .then(done, done.fail);
     });
 
+    it('should not display hover for display: none', function(done) {
```

Comment:
This test fails on CircleCI. Does it pass on your machine?

Reply:
No it does not and I have problems with the tests in the first place on my machine, since its a windows.
But yes, that should work.

Reply:
The test is not passing.
Please simply replace it with 
```js
it('should not display hover for display: none', function(done) {
        Plotly.newPlot(gd, {
            data: [{
                name: 'A',
                y: [1]
            }, {
                name: 'B',
                y: [2],
                hoverinfo: 'none'
            }],
            layout: {
                hovermode: 'x unified',
                showlegend: false,
                width: 500,
                height: 500,
                margin: {
                    t: 50,
                    b: 50,
                    l: 50,
                    r: 50
                }
            }
        })
        .then(function() {
            _hover(gd, { xpx: 200, ypx: 200 });
            assertLabel({title: '0', items: [
                'A : 1'
            ]});
        })
        .then(done, done.fail);
    });
```